libbeat/dashboards/import_dashboards.go

```diff
@@ -105,6 +107,9 @@ func (cl *CommandLine) Read() error {
 	if cl.opt.Url == "" && cl.opt.File == "" && cl.opt.Dir == "" {
 		return errors.New("ERROR: Missing input. Please specify one of the options -file, -url or -dir")
 	}
+	if cl.opt.Beat == "" {
+		cl.opt.Beat = beat
+	}
```

Comment:
I think it would be better to set it to `beat` only if the option is not used. This way, even when using the version from a Beat package, one can force importing all dashbaords with `-beat ""`.